components/user/onboarding/UserInfoForm.tsx

```diff
@@ -99,6 +99,9 @@ const UserInfoForm = ({ userDetails }: UserInfoFormProps) => {
             label="English Level"
             dataSet={ENGLISH_LEVELS.map((level) => ({ id: level.value, title: level.label }))}
             initialValue={userDetails.englishLevel ?? null}
+            clearOnFocus={true}
+            showClear={false}
+            showChevron={false}
```

Comment:
Why do we not show a chevron, I think it is a good indicator to tell this is a dropdown.

Reply:
I can put it back in, but the issue is that if the field has an initial value, then when the chevron is tapped the dropdown only shows that one value (since the dropdown items are filtered by what is written on the field). Instead, now the user would decide to tap on somewhere on the field, the field would get cleared, and all the items would be shown on the dropdown. 

It's your call, what do you think?

Reply:
Damn, that is faulty behavior by the component. Anyways, let's just leave it this way then.